views/page-tou.pug

```diff
@@ -30,10 +30,22 @@ block content
 			li Encouraging/glorifying harm or self-harm of others, including malicious, directed use of "die", "kys", etc. If you or someone you know is at risk of self-harm, #[a(href="https://en.wikipedia.org/wiki/List_of_suicide_crisis_lines") please seek help from a professional or the authorities.]
 			h4
 				strong Explicitly Forbidden Words:
-			li N word -- likely ban on sight
-			li F slur -- likely timeout on sight
-			li &quot;R*tard&quot;
-			li Words that are equivalent to those listed above (e.g., &quot;libtard&quot; for &quot;r*tard&quot;).
+			li You can see the
+				em  uncensored
+				span  word by clicking on the "Click to see"
+			br
+			li N word -- likely ban on sight --
+				p(class="revealer" onclick="toggle('nword')") Click to see:
+				p(class="hide" id="nword" style="display: none") nigger
```

Comment:
I don't love the uncensored version of this one being present fully at all, there are decent odds that it'll find itself indexed or similar.